apps/web/src/components/templates/email-editor/EmailInboxContent.tsx

```diff
@@ -44,11 +47,12 @@ export const EmailInboxContent = ({
             <Controller
               name={`steps.${index}.template.subject` as any}
               control={control}
-              render={({ field, fieldState }) => {
+              render={({ field }) => {
                 return (
                   <Input
                     {...field}
-                    error={fieldState.error?.message}
+                    required
```

Comment:
Do we need to set all the input fields as required?

Reply:
No, only subject and now layout are required in email editor.  wdyt @p-fernandez ?

Reply:
I thought this was rendering multiple inputs for different fields. Misread.